### PR TITLE
Add missing SVG elements/attributes

### LINKED ISSUE #148: Towards v1.0
I am mopping up the last issues required for the v1.0 release 🚀 

This would be the time to discuss API changes if there is something we are still not sure about. 

# Feliz issues
I would like to fix these issues in Feliz before going stable
 - [x] [Clean up memo overloads](https://github.com/Zaid-Ajaj/Feliz/issues/118)
 - [x] [Add KeyboardEvent goodness to easily bind to key press events](https://github.com/Zaid-Ajaj/Feliz/issues/93)
 - [x] [Finish React API bindings](https://github.com/Zaid-Ajaj/Feliz/issues/127)
 - [x] [Review prop.ref API implementation and add examples](https://github.com/Zaid-Ajaj/Feliz/issues/147)
 - [x] [Finish React hook bindings](https://github.com/Zaid-Ajaj/Feliz/issues/50)
 - [x] [Finish SVG element and attribute bindings](https://github.com/Zaid-Ajaj/Feliz/issues/184)

# Ecosystem

I already bumped Feliz bindings to v1.0 like Recharts, PigeonMaps, Popover and the template. I would like to get [Feliz.MaterialUI](https://github.com/cmeeren/Feliz.MaterialUI) to stable version as well cc @cmeeren. Same for [Feliz.Plotly](https://github.com/Shmew/Feliz.Plotly) cc @Shmew 

- [x] Stable Feliz.MaterialUI
- [x] Stable Feliz.Plotly 

# Final nice to haves

I would really appreciate it if we could also work on [Feliz.AntDesign](https://github.com/Zaid-Ajaj/Feliz.AntDesign) and get it into a nice form for the the stable release thought it will require a lot of work. Not a blocker for Feliz v1.0 but would be great to have along with the other ecosystem libraries.

### LINKED ISSUE #174: Can't set IFrame and SVG attributes
Trying to implement Youtube Videos and SVGs on a website, yet can't set some properties:
```
            Html.iframe [ 
              prop.className "videostream"
              prop.src v.url
              prop.allowFullScreen true // Missing Feature in Feliz?
              prop.scrolling "no" // Missing Feature in Feliz?
              prop.frameBorder 0 // Missing Feature in Feliz?

```

```
                  Html.svg [ 
                    prop.className "fill-current h-3 w-3" ; 
                    Html.svg.viewBox "0 0 20 20" // Missing Feature in Feliz?
                    Html.svg.xmlns "http://www.w3.org/2000/svg" // Missing Feature in Feliz?
                    prop.children [ 
                      Html.path [ 
                        Html.d "M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" // Missing Feature in Feliz?
                      ]
                    ]
                  ]
```

### LINKED ISSUE #184: Finish SVG elements and attributes
It [appears](https://github.com/Zaid-Ajaj/Feliz/issues/174) not all of the SVG elements and attributes are present in the library. We should finish them.

### PR DESCRIPTION
**Pull request by Shmew**
Helps further fix #174, resolves #184, and potentially resolves #148.

### CODE REVIEW

File: Feliz/Properties.fs [hunk 1]
```diff
@@ -38,7 +39,67 @@ type AriaRelevant =
     /// Text is added to any DOM descendant nodes of the live region.
     | Text
 
+[<RequireQualifiedAccess; EditorBrowsable(EditorBrowsableState.Never)>]
+module PropHelpers =
```
Comment (Zaid-Ajaj) [inline Feliz/Properties.fs:43]: Please add `internal` modifier into this module?

Comment (Shmew) [reply Feliz/Properties.fs:43]: We can't use `internal` since the `prop` methods are inlined.

File: Feliz/Properties.fs [hunk 2]
```diff
@@ -911,20 +1331,78 @@ type prop =
...
-    /// SVG attribute to define the radius of a circle
+    /// SVG attribute to define the radius of a circle.
     static member inline r (value: int) = Interop.mkAttr "r" value
 
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (value: float) = Interop.mkAttr "radius" value
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (value: int) = Interop.mkAttr "radius" value
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (xRadius: float, yRadius: float) = Interop.mkAttr "radius" (xRadius, yRadius)
```
Comment (Zaid-Ajaj) [inline Feliz/Properties.fs:1398]: When the `radius` property takes two numbers as input, more specifically as [Number-optional-number](https://developer.mozilla.org/en-US/docs/Web/SVG/Content_type#Number-optional-number) the the values should be serialized to a comma-separated string:
```fs
static member inline radius (xRadius: float, yRadius: float) = 
    Interop.mkAttr "radius" (unbox<string> xRadius  + "," + unbox<string> yRadius)
```

File: Feliz/Properties.fs [hunk 3]
```diff
@@ -1042,6 +1567,19 @@ type prop =
     /// Defines the first number if other than 1.
     static member inline start (value: string) = Interop.mkAttr "start" value
 
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (value: float) = Interop.mkAttr "stdDeviation" value
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (value: int) = Interop.mkAttr "stdDeviation" value
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (xAxis: float, yAxis: float) = Interop.mkAttr "stdDeviation" (xAxis, yAxis)
```
Comment (Zaid-Ajaj) [inline Feliz/Properties.fs:1575]: Same as `radius` property, should not output a tuple but instead serialize the values as `string` into the value

File: Feliz/Properties.fs [hunk 4]
```diff
@@ -1139,6 +1815,45 @@ type prop =
...
+    /// The values attribute has different meanings, depending upon the context where itʼs used, 
+    /// either it defines a sequence of values used over the course of an animation, or itʼs a 
+    /// list of numbers for a color matrix, which is interpreted differently depending on the 
+    /// type of color change to be performed.
+    static member inline values (values: seq<string>) = Interop.mkAttr "values" (values |> String.concat " ")
+
+    /// Defines the position and dimension, in user space, of an SVG viewport. 
+    static member inline viewBox (minX: int, minY: int, width: int, height: int) =
+        Interop.mkAttr "viewBox" (
+            (string minX) + " " +
```
Comment (Zaid-Ajaj) [inline Feliz/Properties.fs:1852]: Please use `unbox<string>` to avoid the runtime operation

**Pull request merged**